presto-tests/src/main/java/io/prestosql/tests/AbstractTestJoinQueries.java

```diff
@@ -757,12 +757,69 @@ public void testFullJoinWithCoalesce()
                         "FULL OUTER JOIN (VALUES 1, 4, null) u(a) ON t.a = u.a " +
                         "GROUP BY 1",
                 "VALUES (1, 1), (2, 1), (3, 1), (4, 1), (40, 1), (100, 2)");
+
+        assertQuery(
```

Comment:
does this test for `constants` arguments?

Reply:
The third query does.